### PR TITLE
Spaces hexagon images

### PR DESCRIPTION
**Pull request by mikeger**
- Added spaces images.
- Using the clipping `mask` layer.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/ConversationList/ConversationListTopBar.swift [hunk 1]
```diff
@@ -63,26 +65,51 @@ final class ConversationListTopBar: UIView {
...
-            constrain(self, new) { selfView, new in
-                new.center == selfView.center
+            constrain(middleViewContainer, new) { middleViewContainer, new in
+                new.edges == middleViewContainer.edges
             }
         }
     }
     
-    public let separatorLineView = OverflowSeparatorView()
+    public var overflowSeparatorView: Bool = true {
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/ConversationList/ConversationListTopBar.swift]: From the naming this sounds like it is a `UIView`, might make sense to rename it to `showsSeparatorView` or something like that.

**Comment by mikeger** [comment]
@daehn updated!

**Comment by mikeger** [comment]
@daehn I added joining the space with link, please check

File: WireExtensionComponents/Icons/Autogenerated/WireStyleKit.h [hunk 2]
```diff
@@ -18,17 +18,20 @@
 
 #import <UIKit/UIKit.h>
 
-typedef NS_ENUM(NSInteger, WireStyleKitResizingBehavior)
+
+
+typedef enum : NSInteger
```
Comment (typfel) [inline WireExtensionComponents/Icons/Autogenerated/WireStyleKit.h]: Why removing `NS_ENUM`? Doesn't it provide some hints for the compiler?

Comment (daehn) [inline WireExtensionComponents/Icons/Autogenerated/WireStyleKit.h]: It's autogenerated, so maybe PaintCode changed something under their hood

Comment (mikeger) [inline WireExtensionComponents/Icons/Autogenerated/WireStyleKit.h]: Right, PaintCode stuff.

Comment (typfel) [inline WireExtensionComponents/Icons/Autogenerated/WireStyleKit.h]: 🎨📄

**Pull request merged**